src/document.c

```diff
@@ -825,7 +825,8 @@ GeanyDocument *document_new_file_if_non_open(void)
 }
 
 
-/**
+/** @girskip
```

Comment:
document_open_file() isn't considered a constructor. Alternatively we can make it one (using the constructor annotation) to enforce correspondence with document_new_file(). But since they aren't true constructors I prefer my proposed change. Unfortunately, there is no annotation to enforce non-constructor behavior.